src/AspNetCore.Identity.MongoDB/MongoIdentityBuilder.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace AspNetCore.Identity.MongoDB
+{
+    public class MongoIdentityBuilder
+    {
+        public MongoIdentityBuilder(Type user, IServiceCollection services)
+        {
+            UserType = user;
+            Services = services;
+        }
+
+        public Type UserType { get; private set; }
+        public IServiceCollection Services { get; private set; }
+
+        private MongoIdentityBuilder AddScoped(Type serviceType, Type concreteType)
+        {
+            Services.AddScoped(serviceType, concreteType);
+            return this;
+        }
+
+        public virtual MongoIdentityBuilder AddUserValidator<T>() where T : class
```

Comment:
This doesn't seem to be used and several other members of this class. Why do we need this (several other similar ones) as a public member?